### ISSUE TITLE
COG kopiowanie emaila usera

### ISSUE THREAD

**Issue by bartsman** [opencaching/opencaching-pl#2378]
W widoku zarządzania userem, chciałbym obok przycisku do wysyłania emaila mieć przycisk do kopiowanie tego emaila.

**Comment by bartsman** [comment]
Task completed and uploaded to the server.
Needs adjustments in the future but can stay as is for now.

**Comment by deg-pl** [comment]
@bartsman Good idea is to add something like "fixes #2378" in PR (see https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue).
When PR is merged, issue will be auto closed then.
